neo4j/api.py

```diff
@@ -220,14 +243,12 @@ def version_info(self):
                 pass
         return tuple(value)
 
-    def _update_metadata(self, metadata):
-        """Internal, update the metadata and perform check that the prefix is whitelisted by calling self.version()
-
-        :param metadata: metadata from the server
-        :type metadata: dict
+    def update(self, metadata):
```

Comment:
Why not internal anymore?

Reply:
This was being called from separate modules (Bolt 3 and Bolt 4 handler modules). Such trans-module calls are not considered internal and this was therefore unpythonic. The IDE highlights it as a bad code smell.